README.md

```diff
@@ -40,6 +40,13 @@ Vim to load it for certain files. Just add something like this to your vimrc:
     augroup END
 ~~~
 
+For vimwiki users, use
+
+~~~ vim
+au FileType vimwiki set syntax=markdown.pandoc
```

Comment:
```suggestion
augroup pandoc_syntax
  autocmd! FileType vimwiki set syntax=markdown.pandoc
augroup END
```

Please wrap this in an auto-command group ... because *everything* should be and when providing snippets like this people forget that and then send us bug reports because it does something unexpected.